Samsung/WATT
# Do not add 'Live Preview' button for remote samples

[Issue] http://suprem.sec.samsung.net/jira/browse/TIZENWF-2660
[Problem] Not needed icon, live preview feature is handled by DE
[Solution] Do not add 'Live Preview' icon if project is opened from
    remote sample.
[TEST 1]
    1. Open http://localhost:3000/demos/?path=mobile%2FUIComponents%2Fcomponents%2Fcontrols%2Fcheckbox.html
    2. 'Live Preview' button should *not* be visible.

[TEST 2]
    1. Open http://localhost:3000/
    2. Login and create sample web application.
    3. Open the project.
    4. 'Live Preview' button should be visible.

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>
